src/jit/flowgraph.cpp

```diff
@@ -1815,7 +1815,7 @@ void Compiler::fgComputeReachabilitySets()
 
     for (block = fgFirstBB; block != nullptr; block = block->bbNext)
     {
-        // Initialize the per-block bbReach sets. (Note that we can't just call BlockSetOps::ClearD()
```

Comment:
I think this comment is incorrect. I'm fine with leaving the code as-is, but at least the comment should be fixed.